### DISCUSSION TITLE
Broken Dependency

### DISCUSSION THREAD

**Discussion by 3FLLC** [bgrabitmap/bgracontrols#139] [General]
I am just learning how LCL components plug-in.

I tried to install the BGRA Controls, received the above message. However, nothing in the graph of StringList of the left have a RED X, a couple have a Green +, and do not say installed.

Is there an easier way to find what package(s) are missing and where to download them? I am chasing this for 1 simple effect, slightly gradient buttons - to convey a modern look to my Lazarus project.

**Comment by lainz** [comment]
Hi, use Online Package Manager to install
https://wiki.lazarus.freepascal.org/Online_Package_Manager
